examples/nucleo_f439zi/can_m2515/main.cpp

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2013, Kevin Läufer
+ * Copyright (c) 2013-2017, Niklas Hauser
+ * Copyright (c) 2016, Raphael Lehmann
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <modm/board.hpp>
+#include <modm/platform/spi/spi_master_1.hpp>
+#include <modm/platform/spi/spi_master_2.hpp>
+#include <modm/processing/protothread.hpp>
+#include <modm/driver/can/mcp2515.hpp>
+
+#define SENDER 1
```

Comment:
Please create two example instead of using a define and `#ifdef` in the code, that is not even overwritable from command line.

Having two examples is the only way to have all code checked by CI...

Reply:
done, i have remove the example and made it a little more generic, so that the example acts as both sender/receiver at the same time